<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fightscene</class>
 <widget class="QWidget" name="Fightscene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="attackButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:rgba(255,255,255,0.8);}
QPushButton:hover{color: red;}</string>
   </property>
   <property name="text">
    <string>普通攻击</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="magicAttackButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>550</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:rgba(255,255,255,0.8);}
QPushButton:hover{color: royalblue;}</string>
   </property>
   <property name="text">
    <string>魔法攻击</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="surrenderButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>550</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:rgba(255,255,255,0.8);}
QPushButton:hover{color: white;}</string>
   </property>
   <property name="text">
    <string>投降</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="rivalHealthBar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>561</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar{background:rgba(255, 255, 255, 128); border:1px solid; border-color:grey} QProgressBar::chunk{background:red}</string>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="playerHealthBar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>370</y>
     <width>561</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar{background:rgba(255, 255, 255, 128); border:1px solid; border-color:grey} QProgressBar::chunk{background: lime;}
</string>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Impact</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: lime</string>
   </property>
   <property name="text">
    <string>HP</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Impact</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red</string>
   </property>
   <property name="text">
    <string>HP</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="playerPic">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: solid;
border-color: lime;
border-width: 2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="rivalPic">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>110</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: solid;
border-color: red;
border-width: 2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="playerName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>玩家名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="rivalName">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>70</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>敌方名称</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="rivalHP">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Impact</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red</string>
   </property>
   <property name="text">
    <string>100/100</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="playerHP">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>370</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Impact</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: lime</string>
   </property>
   <property name="text">
    <string>100/100</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="effectLabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:blue</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="roundText">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Let's go Digital</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 0)</string>
   </property>
   <property name="text">
    <string>00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 0)</string>
   </property>
   <property name="text">
    <string>ROUND</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="playerMagicBar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>390</y>
     <width>561</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar{background:rgba(255, 255, 255, 128); border:1px solid; border-color:grey} QProgressBar::chunk{background: DeepSkyBlue;}
</string>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="playerMP">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>390</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Impact</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: DeepSkyBlue</string>
   </property>
   <property name="text">
    <string>100/100</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Impact</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: DeepSkyBlue</string>
   </property>
   <property name="text">
    <string>MP</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="defendButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>550</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:rgba(255,255,255,0.8);}
QPushButton:hover{color: yellow;}</string>
   </property>
   <property name="text">
    <string>防御</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dodgeButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>550</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:rgba(255,255,255,0.8);}
QPushButton:hover{color: green;}</string>
   </property>
   <property name="text">
    <string>闪避</string>
   </property>
  </widget>
  <widget class="QListWidget" name="infoList">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>420</y>
     <width>701</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 128);
border-style: solid;
border-width: 1px;
border-color: grey;</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>31</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 128);
border-style: solid;
border-width: 1px;
border-color: grey;</string>
   </property>
   <property name="text">
    <string>战斗记录</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="skillList">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>420</y>
     <width>701</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color: rgba(0, 0, 0, 128);
border-style: solid;
border-width: 1px;
border-color: grey;</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="flow">
    <enum>QListView::LeftToRight</enum>
   </property>
   <property name="resizeMode">
    <enum>QListView::Fixed</enum>
   </property>
   <property name="gridSize">
    <size>
     <width>60</width>
     <height>121</height>
    </size>
   </property>
   <property name="uniformItemSizes">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="skillLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>31</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color: rgba(0, 0, 0, 128);
border-style: solid;
border-width: 1px;
border-color: grey;</string>
   </property>
   <property name="text">
    <string>魔法技能</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="skillButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>550</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:rgba(255,255,255,0.8);}
QPushButton:hover{color: blue;}</string>
   </property>
   <property name="text">
    <string>技能</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="playerEffectLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="rivalEffectLabel">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>110</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="skillDesc">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>711</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: rgb(0, 0, 0, 128);
border-style: solid;
border-width: 2px;
border-color: grey;</string>
   </property>
   <property name="text">
    <string>技能描述</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>attackButton</zorder>
  <zorder>magicAttackButton</zorder>
  <zorder>surrenderButton</zorder>
  <zorder>rivalHealthBar</zorder>
  <zorder>playerHealthBar</zorder>
  <zorder>label_2</zorder>
  <zorder>label_1</zorder>
  <zorder>playerPic</zorder>
  <zorder>rivalPic</zorder>
  <zorder>playerName</zorder>
  <zorder>rivalName</zorder>
  <zorder>rivalHP</zorder>
  <zorder>playerHP</zorder>
  <zorder>effectLabel</zorder>
  <zorder>roundText</zorder>
  <zorder>label_3</zorder>
  <zorder>playerMagicBar</zorder>
  <zorder>playerMP</zorder>
  <zorder>label_4</zorder>
  <zorder>defendButton</zorder>
  <zorder>dodgeButton</zorder>
  <zorder>infoList</zorder>
  <zorder>infoLabel</zorder>
  <zorder>skillList</zorder>
  <zorder>skillLabel</zorder>
  <zorder>skillButton</zorder>
  <zorder>playerEffectLabel</zorder>
  <zorder>rivalEffectLabel</zorder>
  <zorder>skillDesc</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
